src/test/config.sol

```diff
@@ -1971,10 +1971,10 @@ contract Config {
             offboarding:  false
         });
         afterSpell.collaterals["GNO-A"] = CollateralValues({
-            aL_enabled:   true,
-            aL_line:      5 * MILLION,
-            aL_gap:       3 * MILLION,
-            aL_ttl:       8 hours,
+            aL_enabled:   false,
```

Comment:
Why did this get added to the content?

Reply:
As we discussed before in spell `2024-04-26` setting DC to 0 is not enough if we have AL enabled. As you sad before it on of those technical detail that should be done by us implicitly